src/components/Message/MessageErrorText.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import { StreamMessage, useTranslationContext } from '../../context';
+import { DefaultStreamChatGenerics } from '../../types/types';
+import { isMessageBounced } from './utils';
+
+export interface MessageErrorTextProps<
+  StreamChatGenerics extends DefaultStreamChatGenerics = DefaultStreamChatGenerics
+> {
+  message: StreamMessage<StreamChatGenerics>;
+  theme: string;
+}
+
+export function MessageErrorText<
+  StreamChatGenerics extends DefaultStreamChatGenerics = DefaultStreamChatGenerics
+>({ message, theme }: MessageErrorTextProps<StreamChatGenerics>) {
+  const { t } = useTranslationContext('MessageText');
+
+  if (message.type === 'error' && !isMessageBounced(message)) {
+    return (
+      <div className={`str-chat__${theme}-message--error-message str-chat__message--error-message`}>
+        {t<string>('Error · Unsent')}
+      </div>
+    );
+  }
+
+  if (message.status === 'failed') {
+    return (
+      <div className={`str-chat__${theme}-message--error-message str-chat__message--error-message`}>
+        {message.errorStatusCode !== 403
+          ? t<string>('Message Failed · Click to try again')
+          : t<string>('Message Failed · Unauthorized')}
```

Comment:
I think this key is missing translations.

Reply:
Both keys are there as far as I can see - this part was not changed BTW, just extracted to a separate component, because the logic was becoming kinda complex.

Reply:
My bad, did not realize